main.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'app'
+
+def call_app(app, choice)
+  case choice
+  when 1
+    app.list_books
+  when 2
+    app.list_people
+  when 3
+    app.create_person
+  when 4
+    app.create_book
+  when 5
+    app.create_rental
+  when 6
+    app.list_rentals
```

Comment:
- [ ] Kindly fix this error in the `6 - List all rentals for a given person id ` case
![image](https://user-images.githubusercontent.com/24703171/194209322-1ca6e4cd-5061-4e69-9e80-ac046968a3d8.png)